app/src/main/res/values/strings.xml

```diff
@@ -8,6 +8,7 @@
     <string name="preference_id">ProtoManTakeMeByTheHand</string>
     <string name="is_possum_enabled">possum_enabled</string>
     <string name="authenticated_by_possum">authenticated_by_possum</string>
+    <string name="should_check_possum">should_check_possum</string>
```

Comment:
We could consider creating a `PossumStore` file for all these static strings, since they're never translated, and also just have some neat helper functions to get the things we need. Don't need to add it to this PR, though.

Reply:
Yeah I agree. Will do that for the "SDKify" task